bin/p2-replicate/main.go

```diff
@@ -77,6 +77,13 @@ func main() {
 		log.Fatalf("Invalid manifest: %s", err)
 	}
 
+	for _, host := range *hosts {
+		_, _, err := store.Pod(kp.RealityPath(host, "p2-preparer"))
+		if err != nil {
+			log.Fatalf("Could not verify preparer on %s: %s", host, err)
```

Comment:
Most people won't understand what this means - I'd suggest `"p2 does not appear to be running on host %s"`